js/tidelinedata.js

```diff
@@ -335,11 +335,13 @@ function TidelineData(data, opts) {
             d.displayOffset = 0;
           }
           else if (d.type === 'message') {
-            var datumDt = new Date(d.time);
-            var offsetMinutes = datumDt.getTimezoneOffset();
-            datumDt.setUTCMinutes(datumDt.getUTCMinutes() - offsetMinutes);
-            d.normalTime = datumDt.toISOString();
-            d.displayOffset = 0;
+            if (dt.isATimestamp(d.time)) {
```

Comment:
Yeah, it's a bit complicated. If it's not a valid date, we return false, then we just *don't* generate a `normalTime` and a `displayOffset` (because there's no `else` clause), and then the datum gets removed during tideline's validation (because both `normalTime` and `displayOffset` are required). Basically the whole `isATimestamp` is just a fancy replacement for the error-catching that's near impossible with JavaScript Date, so we don't choke on the error.